bin/update_config.sh

```diff
@@ -18,5 +18,9 @@ for CLUSTER in ${CLUSTERS:=iowa-a}; do
         fi
     done
 done
+
+TEST_IMAGE=mozorg/bedrock_test:${GIT_COMMIT}
+sed -i -e "s|TEST_IMAGE: .*|TEST_IMAGE: ${TEST_IMAGE}|;s|image: mozorg/bedrock_test.*|image: ${TEST_IMAGE}|" .gitlab-ci.yml
+
```

Comment:
This is from #8218 right?

Reply:
right, I based this on #8218 and will rebase if that one merges first, or we can close it out in favor of this one if we want to just merge this.